assets/sass/06-components/navigation.scss

```diff
@@ -230,37 +230,43 @@
 				display: none;
 			}
 
-			// When the sub-menu is closed, display the plus icon
-			&[aria-expanded="false"] {
-				/* stylelint-disable */
-				background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' width='18' height='18'><path d='M18 11.2h-5.2V6h-1.6v5.2H6v1.6h5.2V18h1.6v-5.2H18z' fill='%2328303d'/></svg>") no-repeat center center;
-				.is-background-dark & {
-					background: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' width='18' height='18'><path d='M18 11.2h-5.2V6h-1.6v5.2H6v1.6h5.2V18h1.6v-5.2H18z' fill='%23fff'/></svg>") no-repeat center center;
+			.icon-plus,
+			.icon-minus {
+				height: 100%;
+				display: flex;
+				align-items: center;
```

Comment:
If I remove the `flexbox` + `height` rules, the appearance improves in Safari + Chrome: 

Before: 
![safari](https://user-images.githubusercontent.com/1202812/95619750-16660680-0a3d-11eb-9da7-63bc1c2a3168.png)

After: 
![safari2](https://user-images.githubusercontent.com/1202812/95619761-18c86080-0a3d-11eb-9631-00ff20ddaf96.png)

Unfortunately, this pushes it up too high in Firefox though: 

![ff2](https://user-images.githubusercontent.com/1202812/95619896-5331fd80-0a3d-11eb-977b-f3cc0c4d1555.png)

😩